### PR TITLE
docs(ru): Translate directory features

### PR DESCRIPTION
**Pull request by zavvla**
Added translate "features/rendering-modes" for RU

### CODE REVIEW

**Comment by zavvla** [comment]
@DreaMinder make a review please

File: content/ru/guides/features/rendering-modes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Рендеринг
+description: Рендеринг
+position: 1
+category: features
+---
+
+## Рендеринг на стороне сервера и статические сайты
+
+Рендеринг(SSR) на стороне сервера происходит каждый раз, когда запрашивается страница, поэтому нужен сервер который сможет обсуживать такие запросы.
```
Comment (DreaMinder) [inline content/ru/guides/features/rendering-modes.md:10]:
```suggestion
Рендеринг (SSR) на стороне сервера происходит каждый раз, когда запрашивается страница, поэтому нужен сервер который сможет обслуживать такие запросы.
```

File: content/ru/guides/features/rendering-modes.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Статические сайты очень похожи на приложение(SPA), которые отображаются на стороне сервера, с разницей лишь в том, что статический сайт рендерится во время сборки, поэтому сервер не нужен.
```
Comment (DreaMinder) [inline content/ru/guides/features/rendering-modes.md:12]:
```suggestion
Статические сайты очень похожи на приложения (SPA), которые рендерятся на стороне сервера, с разницей лишь в том, что статический сайт рендерится во время сборки, поэтому сервер не нужен.
```

File: content/ru/guides/features/rendering-modes.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Навигация с одной страницы на другую осуществляется на стороне клиента(в браузере).
```
Comment (DreaMinder) [inline content/ru/guides/features/rendering-modes.md:13]:
```suggestion
Навигация с одной страницы на другую осуществляется на стороне клиента (в браузере).
```

File: content/ru/guides/features/rendering-modes.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+}
+```
+
+<base-alert type="info">
+
+Включать `ssr: true` в конфигурации nuxt.config.js **не нужно**, эта опция включена по умолчанию.
+
+</base-alert>
+
+## Рендеринг на стороне клиента(в браузере)
```
Comment (DreaMinder) [inline content/ru/guides/features/rendering-modes.md:29]:
```suggestion
## Рендеринг на стороне клиента (в браузере)
```

File: content/ru/guides/features/rendering-modes.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Если рендеринг происходит на стороне клиента(в браузере), рендеринг на сервере(SSR) работать не будет.
```
Comment (DreaMinder) [inline content/ru/guides/features/rendering-modes.md:31]:
```suggestion
Если включен режим рендеринга "только-на-клиенте" (в браузере), рендеринг на сервере (SSR) работать не будет.
```

File: content/ru/guides/features/rendering-modes.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+Рендеринг на стороне клиента означает, что содержимое документа будет генерироваться при помощи JavaScript.
+Вместо того, чтобы получить все содержимое HTML с сервера, мы получим базовую страницу HTML и JavaScript. После загрузки JavaScript начнет выполняться в браузере и отобразит результат работы JavaScript.
```
Comment (DreaMinder) [inline content/ru/guides/features/rendering-modes.md:33]:
```suggestion
Вместо того, чтобы получить все содержимое HTML с сервера, мы получим базовую страницу HTML и JavaScript. После загрузки, JavaScript начнет выполняться в браузере и отобразит результат его работы.
```

**Review by DreaMinder — CHANGES_REQUESTED** [review]

**Review by DreaMinder — APPROVED** [review]
Nice

**Pull request merged**